docs/_static/versions.json

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "version": "main",
+        "url": "https://discretize.simpeg.xyz/en/main/"
+    },
+    {
+        "name": "0.10.0 (latest)",
+        "version": "v0.10.0",
```

Comment:
Actually, the `version` field should not have the leading `v` because these versions should be parsable by the compare-versions node module (https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/announcements.html#version-warning-banners).

This way, the warning version banners can behave properly. If we have the `v`, any previous version is recognized as a development version, so the banner shows the wrong text.

We can still use the leading `v` in the url though.

Reply:
`compare_versions` ignores any leading `v`
https://www.npmjs.com/package/compare-versions

Reply:
I actually experienced a misbehaviour in the version banner in simpeg before I changed the versions to not have a leading `v`. But yes, you're right that they seem to support it...

Ignore this then and we'll see if it works well.

Reply:
It looks like the pydata sphinx theme uses the leading `v` in their's for matching, https://github.com/pydata/pydata-sphinx-theme/blob/main/docs/_static/switcher.json

Reply:
Then maybe there was something else that was failing back then... Well, I'm glad we don't have to fix it.